controllers/resource/reconciler_test.go

```diff
@@ -109,6 +110,12 @@ var nutanixWorkerMachineConfigSpec string
 //go:embed testdata/nutanix/etcdMachineConfig.yaml
 var nutanixEtcdMachineConfigSpec string
 
+func getSecret() *corev1.Secret {
```

Comment:
In an offline convo with @g-gaston we decided to duplicate the test function across two test packages rather than use the `internal/test` package I had up.